salt/states/openvswitch_bridge.py

```diff
@@ -12,12 +12,16 @@ def __virtual__():
     return (False, "openvswitch module could not be loaded")
 
 
-def present(name):
+def present(name, parent=None, vlan=None):
     """
     Ensures that the named bridge exists, eventually creates it.
 
     Args:
         name: The name of the bridge.
+        parent: The name of the parent bridge (if the bridge shall be created
```

Comment:
These new additional parameters could use a versionadded see https://docs.saltproject.io/en/latest/topics/development/conventions/style.html
```
.. versionadded:: 3006
```
see above about dropping the 'A', that is 
```
   name : string
     bridge name
```